src/mssql-server/Dockerfile-mssql-server

```diff
@@ -0,0 +1,17 @@
+﻿# Base OS layer: Latest Ubuntu LTS
```

Comment:
Can't think of a reason why this wouldn't work on Linux, but might be worth getting a guinea pig to trial it before merging

Reply:
Ah, yep, I hadn't thought about that, thanks. Would it be possible for you to give it a try?

If you switch to this branch and `docker compose up db`.

If you execute this on the database and then get a result of '1' it's enabled:

```
SELECT fulltextserviceproperty('IsFulltextInstalled')
```

If you run our Admin service which will run the migrations on this branch the free text catalog with column indexes will be added. You should then be able to run a query like

```
SELECT Summary, Title
FROM dbo.Publications
WHERE FREETEXT ((Summary, Title), 'GCSE OR Social OR Outcomes OR Graduate')
```

If you want to go even further and try it out from EF Core there's some quick and easy code examples in the description of this PR which can be pasted into any controller in the Content API.

Thanks! 🙂

Reply:
All looks great!